content/articles/launching-proppy.md

```diff
@@ -0,0 +1,35 @@
+Title: We are Wizards launch a new product
+Date: 2016-03-25
+Short_summary: After wrangling with proposals for a while we productized one of our internal tools.
+Category: Product
+Authors: Vincent, Tom
+
+*Reading time: ~3 minutes.*
+
+At We Are Wizards we send out a fair number of proposals. In our early
```

Comment:
As you might have noticed we didn't post that much on the blog recently - but there's a good reason for that: we were working on a product!

We send out a fair number of proposals...

Is not too great either but maybe a bit better